### PR TITLE
DAF-4473: [Android] When livestream is being archived, progress bar is still active in control center

### PR DESCRIPTION
**Pull request by mvn-hieunguyen-hn**
## Description
[Android] When livestream is being archived, progress bar is still active in control center
### Problem
[Android] When livestream is being archived, progress bar is still active in control center
### Solution
create new flag to check disable seekbar
### What was done (Please be a little bit specific)

- create new flag to check disable seekbar
- create new logic check flag to add duration

### JIRA
https://dw-ml-nfc.atlassian.net/browse/DAF-4473

### JP (and VN) Specs
None

### Figma
None

### API (Only for API implementation)
None

## Screenshot or Video (Before/After)
Before 

https://github.com/dwango-nfc/betterplayer/assets/79908230/140305cd-3465-4590-9992-a928c145218d

 After

https://github.com/dwango-nfc/betterplayer/assets/79908230/bc922524-586b-493f-9bc8-3c5669a29d3f

## Ready for review checklist
Basically, you must check all. (When you ignore any checks, please write each reason.)

- [x] All sections above are properly filled in
- [x] The PR only deals with one functionality/bug
- [ ] The code include code refactoring? If yes, the following sub checks are required.
    - [ ] The changes were tested. So there is no degradation.
    - [ ] The change is not too big. (Reviewers can review it.)
- [ ] Builds and runs on iOS (No new warnings nor new errors)
- [x] Builds and runs on Android (No new warnings nor new errors)

### CODE REVIEW

File: lib/src/configuration/better_player_data_source.dart [hunk 1]
```diff
@@ -76,6 +76,10 @@ class BetterPlayerDataSource {
   ///platform.
   final BetterPlayerBufferingConfiguration bufferingConfiguration;
 
+  //flag determine hide seekbar of notification
+  //fix bug ticket: https://dw-ml-nfc.atlassian.net/browse/DAF-4473
+  final bool? isHideSeekbarNotification;
```
Comment (mvn-thanhluu-hn) [inline lib/src/configuration/better_player_data_source.dart]:
```suggestion
  ///Flag determine hide seekbar of notification. Currently only supported in Android platform.
  ///Fix bug ticket: https://dw-ml-nfc.atlassian.net/browse/DAF-4473
  final bool? isSeekbarOnNotificationDisable;
```

Please use document comment `///` and note this flag only for Android.

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM

@mvn-hieunguyen-hn Thanks!
I have a small suggestion. Please check it 🙏

**Comment by mvn-thanhluu-hn** [comment]
@noboru-i 
Please review this PR 🙏

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt [hunk 2]
```diff
@@ -662,8 +663,7 @@ internal class BetterPlayer(
                     // LIVE
```
Comment (noboru-i) [inline android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt]: I think, this comment is for `isCurrentMediaItemLive`.
How about removing this line?
Please check it 🙏

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt [hunk 3]
```diff
@@ -880,8 +902,10 @@ class BetterPlayerPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
         }
 
         // Will be observed to show notification.
-        private var _notificationParameter: MutableLiveData<NotificationParameter?> = MutableLiveData()
+        private var _notificationParameter: MutableLiveData<NotificationParameter?> =
```
Comment (noboru-i) [inline android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt:906]: Is this diff caused by formatter?
If so, please re-check whether other VN members also cause this diff or not.
(I'm afraid of rollback these changes by other member's format.)

File: lib/src/configuration/better_player_data_source.dart [hunk 4]
```diff
@@ -76,6 +76,10 @@ class BetterPlayerDataSource {
   ///platform.
   final BetterPlayerBufferingConfiguration bufferingConfiguration;
 
+  ///Flag determine hide seekbar of notification. Currently only supported in Android platform.
```
Comment (noboru-i) [inline lib/src/configuration/better_player_data_source.dart]: Sorry.
iOS works OK...?
(It seems related to `isLiveStream`. We cannot use it?)

**Review by noboru-i — CHANGES_REQUESTED** [review]
@mvn-hieunguyen-hn 
I add some comments. Please check it 🙏

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt (hunk 3 above)
Comment (mvn-hieunguyen-hn) [reply android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt:906]: I have checked with other VN members, and they also cause this difference. It's possible that the previous coder forgot to format the code.

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt (hunk 2 above)
Comment (mvn-hieunguyen-hn) [reply android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt]: @noboru-i 
I added a comment for case Live Finished And Making Archive.

File: lib/src/configuration/better_player_data_source.dart (hunk 4 above)
Comment (mvn-hieunguyen-hn) [reply lib/src/configuration/better_player_data_source.dart]: @noboru-i 
When I checked the code, I found this variable, but I didn't see its clear meaning for case `Live Finished And Making Archive`. So, I added a new variable to make the code more explicit. I tried using variable `isCurrentMediaItemLive ` and found that the application worked well, so I deleted the new variable I had added.

**Comment by mvn-hieunguyen-hn** [comment]
@noboru-i 
I have finished fixing your comments, please check them again for me, thanks  🙏

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt (hunk 3 above)
Comment (noboru-i) [reply android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt:906]: OK, Thanks.
I understand. 👍

**Review by noboru-i — APPROVED** [review]
LGTM.
